sycl/include/CL/__spirv/spirv_types.hpp

```diff
@@ -108,13 +108,22 @@ enum class GroupOperation : uint32_t {
   ExclusiveScan = 2
 };
 
+#if (SYCL_EXT_ONEAPI_MATRIX_VERSION > 1)
+enum class MatrixLayout : uint32_t {
+  RowMajor = 0,
+  ColumnMajor = 1,
+  Packed = 2,
+  Dynamic = 3
+};
+#else
 enum class MatrixLayout : uint32_t {
   RowMajor = 0,
   ColumnMajor = 1,
   PackedA = 2,
   PackedB = 3,
   Unused = 4
```

Comment:
As for use, Dmitry is removing unecessary in this patch https://github.com/intel/llvm/pull/7335
it will be merged soon. Once it is merged, this PR should be updated as well.

Reply:
@yubingex007-a11y https://github.com/intel/llvm/pull/7335 has been merged. You can rebase your patch to remove all "unnecessary" use occurrences.